examples/2016-10-31/api_multiple_origin_cors/template.yaml

```diff
@@ -0,0 +1,37 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: 'AWS::Serverless-2016-10-31'
+Description: 'Example of Multiple-Origin CORS using API Gateway and Lambda'
+Resources:
+  ExampleFunction:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      CodeUri: '.'
+      Handler: 'index.handler'
+      Runtime: 'nodejs8.10'
+      Events:
+        GetRoot:
+          Type: 'Api'
+          Properties:
+            Path: '/'
+            Method: 'get'
+        OptionsTest:
+          Type: 'Api'
+          Properties:
+            Path: '/test'
+            Method: 'options'
+        PostTest:
```

Comment:
I see there is some special case for DELETE methods in the function. Is it useful to have a DELETE method in the example API? Up to you.

Reply:
I chose DELETE arbitrarily to demonstrate how to handle preflight requests, since it is one of the methods that is guaranteed to trigger one. So I think it is more a specific example of a general case. 
I do notice now that the event is still named "PostTest", whereas it should maybe be something like "DeleteTest".

Reply:
It says 'PostTest' but has `Method: 'delete`. To really test CORS you should have an OPTIONS method on the /test (or other) path also. Won't this fail a real world CORS scenario as it currently stands? Has you confirmed this works?